### ISSUE TITLE
Testing latest OTTR updates

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/OTTR_Template#470]
Step 1) take a look at the docs here: https://github.com/jhudsl/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots

Please comment on anything that isn't clear (or feel free to make edits to the docs directly)

Step 2) Send the changes to one of your courses. Go here: https://github.com/jhudsl/OTTR_Template/actions/workflows/test-send-updates.yml Then click `Run Workflow` dropdown menu. Type in the name of your repository you'd like to send the changes to. and Click the green `Run workflow` button

Step 3) Comment on any problems you encounter here! Thanks!

**Comment by carriewright11** [comment]
So far reading it quickly - it looks great! maybe adding a sentence to each check about what happens if you put `no` would be helpful? also maybe showing the code for that as well - since this isn't the default?

**Comment by carriewright11** [comment]
also subheadings for spelling check style code, and docker test like the other subheadings might help navigation

**Comment by avahoffman** [comment]
Looks great! A few thoughts below :) 

- Add file preview
- Bulletize end of the intro section
- Clarify a bit of language in render-preview
- Add a screenshot?

-------

# Customizing Automation

By default, all automation steps and checks will run. But depending on the needs of your course, you can turn these on and off by going to the `config_automation.yml` file and switching options to `yes` or `no`. 

✏️ The `config_automation.yml` file looks like this:

```
# Formatting Checks (run at pull request)
check-quizzes: yes
url-checker: yes
render-preview: yes
style-and-sp-check: yes
...
```

✏️ There are two main sets of automation steps and checks run:
- `.github/workflows/pull-request.yml` : run upon opening a pull request
- `.github/workflows/render-all.yml`: run upon changes being merged to the `main` branch 

### render-preview
```
render-preview: yes
```
After you open a pull request, a preview of the renders as they will appear after the pull request is accepted is run and linked to in a comment on the pull request ✏️ _[this sentence was a bit hard to read for me.. maybe rephrase?]_. Upon each commit these previews will re-render and edit the comment with the latest render. These Github Actions are located in `render-preview` section of the `pull-request.yml`.
These previews do NOT incorporate any changes influenced by any changes to the Docker image if the Dockerfile is also updated in the same pull request.

```
spell-check: yes
```
Github actions will automatically [run a spell check on all Rmd's and md's](https://github.com/jhudsl/OTTR_Template/blob/main/.github/workflows/style-and-sp-check.yml) whenever a pull request to the `main` branch is filed.
Depending on your preference, you may find it easier to spell check manually on your local computer before pushing to Github.

It will fail if there are more than 2 spelling errors and you'll need to resolve those before being able to merge your pull request. Errors will be printed out on a GitHub comment on your pull request. 

To resolve those spelling errors, go to this repository's `Actions` tab.
Then, click on the GitHub action from the PR you just submitted.
Scroll all the way down to `Artifacts` and click `spell-check-results`.
This will download a zip file with a TSV that lists all the spelling errors.

✏️ _[maybe add a screenshot here?]

**Comment by cansavvy** [comment]
Thanks for these comments! I've incorporated them! https://github.com/jhudsl/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots

**Comment by carriewright11** [comment]
Looks great! I found one word missing in the style check section that I added - but feel free to rephrase if you wanted something different:
https://github.com/jhudsl/OTTR_Template/wiki/How-to-set-up-and-customize-GitHub-actions-robots/_compare/b515a685b3375f8785779225523a1f7766845243...27eacff35e756df689164fd1ae905168629ffeb0

**Comment by cansavvy** [comment]
This is done!
